widget/widgetAssets/locales/en.json

```diff
@@ -12,17 +12,20 @@
         "EXCLUDELIST_CONFIG_PAGE_SEARCH_PLACEHOLDER": "Search added artifacts",
         "EXCLUDELIST_CONFIG_PAGE_SEARCH_RESULT_LABEL": "Search Result :",
 
+        "EXCLUDELIST_CONFIG_PAGE_VALIDATE_IOC_ERROR_MSG": "Remove invalid indicator values from ",
+
         "EXCLUDELIST_CONFIG_PAGE_BULK_IMPORT_LABEL": "Add indicators manually or use",
         "EXCLUDELIST_CONFIG_PAGE_BULK_IMPORT_LAUNCH_BUTTON": "bulk import",
         "EXCLUDELIST_CONFIG_PAGE_BULK_IMPORT_FILE_IMPORT_BUTTON":"Import",
         "EXCLUDELIST_CONFIG_PAGE_BULK_IMPORT_UPLOADER_LABEL":"Import Indicators",
-        "EXCLUDELIST_CONFIG_PAGE_BULK_IMPORT_UPLOAD_FAILED":"Upload failed. Please try again.",
-        "EXCLUDELIST_CONFIG_PAGE_BULK_IMPORT_FILE_SIZE_EXCEEDED":"File exceeds the maximum size.",
+        "EXCLUDELIST_CONFIG_PAGE_BULK_IMPORT_UPLOAD_FAILED":"Upload failed. Please try again",
```

Comment:
any error to be shown here?

Reply:
No, we cannot fetch the exact error in this case.